cmd/wego/app/cmd.go

```diff
@@ -8,7 +8,17 @@ import (
 )
 
 var ApplicationCmd = &cobra.Command{
-	Use:  "app [subcommand]",
+	Use:   "app [subcommand]",
```

Comment:
See comment above about [subcommand] not being printed and therefore can be removed from here.